soccer/planning/tests/PlannerTest.cpp

```diff
@@ -44,8 +48,31 @@ TEST(Planning, path_target_random) {
                             2,
                             nullptr};
         Trajectory path = planner.plan(std::move(request));
-        assertPathContinuous(path, RobotConstraints{});
+
+        if (path.empty()) {
+            failure_count++;
+            continue;
+        }
+
+        if (!checkPathContinuous(path, RobotConstraints{})) {
+            std::cout << "Saving to /tmp/out.csv" << std::endl;
+            std::ofstream file("/tmp/out.csv");
+            for (auto cursor = path.cursor_begin(); cursor.has_value();
+                 cursor.advance(0.05s)) {
+                auto instant = cursor.value();
+                file << RJ::Seconds(instant.stamp - RJ::now()) << ", "
+                     << instant.position().x() << ", " << instant.position().y()
+                     << ", " << instant.linear_velocity().x() << ", "
+                     << instant.linear_velocity().y() << std::endl;
```

Comment:
What's this file for?

Reply:
Basically just smoke testing the planners, but I had it writing to a file for debugging purposes. I'll go through and clean all of this out.

Reply:
Tests will be fixed in a later PR to mp-rewrite/base.